BUILD.md

```diff
@@ -136,7 +136,8 @@ The steps below will set up a Python virtual environment to run Avalon.
    cd $TCF_HOME/tools/build
    # Create virtual environment directory with name _dev
    python3 -m venv _dev
-   sudo make clean
+   make clean
+   pip3 install wheel
```

Comment:
Another option is to create an insatll target in Makefile and run this command in Makefile target and call make install